code/datums/uplink_item.dm

```diff
@@ -44,7 +44,7 @@ var/list/uplink_items = list()
 	var/only_on_month	//two-digit month as string
 	var/only_on_day		//two-digit day as string
 	var/num_in_stock = 0	// Number of times this can be bought, globally. 0 is infinite
-	var/static/times_bought = 0
```

Comment:
Isn't it used for the Chef DO NOT TIP which HAS to be unique ?

Reply:
`/static/ `means ALL uplink thingies use the same number
Which meant buying 1 megaphone caused you to not be able to purchase the do not tip thingy and that purchasing the latter meant you could only buy 2 megaphones
So basically shitcode
I tested it after my fix, and tested spawning a second uplink, seems to work

Reply:
Nice
I trust you to have it tested